CommandLine/emake/EnigmaCallbacks.cpp

```diff
@@ -0,0 +1,92 @@
+#include "EnigmaCallbacks.hpp"
+
+#include <iostream>
+#include <pthread.h>
+#include <unistd.h>
+
+bool CallBack::_isOutputting = false;
+std::ifstream CallBack::_outFile;
+
+void* CallBack::OutputThread(void*)
+{
+    if (_outFile)
```

Comment:
Two space is my default now in Notepad++ for C++ ok? kthx I use tabs in my own projects though. :smile: